src/main/java/net/minestom/server/item/event/ItemEventHandler.java

```diff
@@ -0,0 +1,33 @@
+package net.minestom.server.item.event;
+
+import net.minestom.server.event.Event;
+import net.minestom.server.event.EventCallback;
+import net.minestom.server.event.handler.EventHandler;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+public class ItemEventHandler implements EventHandler {
+
+    private final Map<Class<? extends Event>, Collection<EventCallback>> eventCallbacks = new ConcurrentHashMap<>();
+    private final Map<String, Collection<EventCallback<?>>> extensionCallbacks = new ConcurrentHashMap<>();
+
+    ItemEventHandler(String namespace) {
+
+    }
+
+    @NotNull
+    @Override
+    public Map<Class<? extends Event>, Collection<EventCallback>> getEventCallbacksMap() {
+        return eventCallbacks;
+    }
+
+    @NotNull
+    @Override
+    public Collection<EventCallback<?>> getExtensionCallbacks(String extension) {
```

Comment:
This could have an overloaded method to take an extension or a discovered extension

Reply:
Probably should be part of EventHandler instead.